package.json

```diff
@@ -25,18 +25,18 @@
     "lint": "prettier --write . && eslint . --fix",
     "release": "zotero-plugin release",
     "test": "echo \"Error: no test specified\" && exit 1",
-    "update-deps": "npm update --save"
```

Comment:
Any reason why npx?

Reply:
I wrongly understood what the save option of the npm update command does, have reverted.